pkg/function-manager/handlers.go

```diff
@@ -139,6 +143,12 @@ func runModelToEntity(m *models.Run, f *functions.Function) *functions.FnRun {
 	} else {
 		secrets = append(secrets, m.Secrets...)
 	}
+	services := f.Services
+	if services == nil {
```

Comment:
Checking if `services == nil` when setting empty slice in CLI (see my other comment).